TACC/Core-Portal
# FP-1016 Disabled Trash button for files in .Trash.

## Overview: ##
Trash button disabled for files in .Trash.
## Related Jira tickets: ##

* [FP-1016](https://jira.tacc.utexas.edu/browse/FP-1016)

## Summary of Changes: ##
Passed path prop in DataFiles component to DataFilesToolbar component, then created an inTrash variable in the DataFilesToolbar component as an additional modifier for the canTrash bool.

## Testing Steps: ##
1. Navigate to .Trash in any file system.
2. Select file(s) in .Trash.
3.  Trash button should be disabled (cannot interact).

## UI Photos:
![image](https://user-images.githubusercontent.com/43251554/132580601-b07f18cc-76a5-41ff-b7ad-bed3f990a464.png)

## Notes: ##
